.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Yeah, intentional. VS Code is pretty nice. Not everything is better compared to WebStorm. TypeScript support is better slightly. Mocha integration and debugging are worse.